src/ray/raylet/node_manager.cc

```diff
@@ -1220,7 +1324,8 @@ void NodeManager::FinishAssignedTask(Worker &worker) {
     actor_notification->driver_id = JobID::nil().binary();
     actor_notification->node_manager_id =
         gcs_client_->client_table().GetLocalClientId().binary();
-    RAY_LOG(DEBUG) << "Publishing actor creation: " << actor_id;
+    RAY_LOG(DEBUG) << "Publishing actor creation: " << actor_id
+                   << "driver_id: " << driver_id;
```

Comment:
additional space before driver_id, otherwise the actor_id and driver_id will be a single word.

Reply:
thanks.